Lib/test/test_ntpath.py

```diff
@@ -856,6 +857,21 @@ def test_nt_helpers(self):
             self.assertIsInstance(b_final_path, bytes)
             self.assertGreater(len(b_final_path), 0)
 
+    @unittest.skipIf(sys.platform != 'win32', "Can only test junctions with creation on win32.")
+    def test_isjunction(self):
+        with os_helper.temp_dir() as d:
+            with os_helper.change_cwd(d):
+                os.mkdir('tmpdir')
+
+                # create a junction via subprocess, since we don't currently
+                # support making them via python directly
+                subprocess.check_call('mklink /J testjunc tmpdir', shell=True)
```

Comment:
`CreateJunction()` is implemented for internal tests.

```suggestion
                import _winapi
                try:
                    _winapi.CreateJunction(tmpdir, 'testjunc')
                except OSError:
                    raise unittest.SkipTest('creating the test junction failed')
```

`os_helper.temp_dir()` creates the directory in the user's temp directory. This should be on the system volume, which has to be NTFS and thus support reparse points. But if for some reason creating the junction fails, the test should just be skipped.